infra/resources/_modules/web_apps/etl_func.tf

```diff
@@ -17,13 +17,15 @@ module "etl_func" {
   health_check_path = "/api/health"
 
   app_settings = {
-    NODE_ENV                       = "production",
-    FUNCTIONS_WORKER_RUNTIME       = "node",
-    MESSAGE_CONTENT_STORAGE_URI    = var.app_settings.message_content_storage_uri
-    EVENTHUB_CONNECTION_URI        = var.app_settings.eventhub_connection_uri,
-    MESSAGE_CONTENT_CONTAINER_NAME = "message-content",
-    MESSAGE_EVENTHUB_NAME          = "io-p-itn-com-etl-messages-evh-01"
-    PDV_TOKENIZER_API_KEY = "@Microsoft.KeyVault(VaultName=${var.common_key_vault.name};SecretName=func-elt-PDV-TOKENIZER-API-KEY)"
+    NODE_ENV                               = "production",
+    FUNCTIONS_WORKER_RUNTIME               = "node",
+    MESSAGE_CONTENT_STORAGE_URI            = var.app_settings.message_content_storage_uri
+    MESSAGE_EVENTHUB_CONNECTION_URI        = var.app_settings.eventhub_connection_uri,
+    MESSAGE_STATUS_EVENTHUB_CONNECTION_URI = var.app_settings.eventhub_connection_uri,
```

Comment:
if the value is the same, why we have two different variables?

Reply:
It can be useful to use two different variables here because we could use two different event hub easily (for example for testing purpose). If you think that it is better to use just one var I can refactor it.

Reply:
now we are using only one event hub namespace, so I would declare only one variable!